README.md

```diff
@@ -75,5 +75,19 @@ custom:
     enhance_print: true
 ```
 
+### Function Scope Configuration
+You can configure lumigo behavior for individual functions as well - 
```

Comment:
Why `as well`? From the code change, it looks like this change only works on the function level, not on the global level?

Reply:
It's under the `Configuration` section, which describes how to configure different attributes Lumigo on the `service` scope.  
This section is meant to describe the different (future?) configuration on the function scope